### ISSUE TITLE
[UIKit] Tracking

### ISSUE THREAD

**Issue by DmitryBespalov** [safe-global/safe-ios#727]

**Comment by liliya-soroka** [comment]
Reopened:
1. no safe screens are tracked when the safe was added
Steps:
1. I have added the safe from the safe settings page ( import key was added before the safe)
2. Open edit safe name
3. Open Advanced safe settings
Current result: screen_settings_no_safe is tracked
![image.png](https://images.zenhubusercontent.com/5ed757c29459bb36bacd4bba/51198f4b-1524-43bf-a7c3-41b00ce03870)
4. Open assets
Current result: screen_assets_no_safe is tracked
![image.png](https://images.zenhubusercontent.com/5ed757c29459bb36bacd4bba/4c8da563-15a0-43c6-8a3f-f003cfa19728)
app version 2.7.0 (227)
5. Open transactions:
Current result: screen_transactions_no_safe is tracked 
![image.png](https://images.zenhubusercontent.com/5ed757c29459bb36bacd4bba/55765d9b-11c5-443b-b9ea-9b4f859a1655)
